docs/pages/web/iconbutton.js

```diff
@@ -63,7 +63,7 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
           },
           {
             name: 'iconColor',
-            type: `"darkGray" | "gray" | "red" | "white"`,
+            type: `"darkGray" | "gray" | "red" | "white" | "brandPrimary"`,
```

Comment:
This fix should really be a separate PR in case this one needs to be reverted for any reason

Feel free to merge this one once it's ✅ 
https://github.com/pinterest/gestalt/pull/2899